### PR TITLE
Bug 4198/v6

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/4198

suricata-verify-pr: 449

### CODE REVIEW

**Comment by inashivb** [comment]
Test failures are because s-v PR is not rebased against the latest master..

File: rust/src/dcerpc/dcerpc.rs [hunk 1]
```diff
@@ -618,6 +620,10 @@ impl DCERPCState {
                     tx.req_lost = true;
                 }
                 tx.req_done = true;
+                if self.flow.is_some() {
```
Comment (victorjulien) [inline rust/src/dcerpc/dcerpc.rs:623]: can't this be done cleaner? `if let Some(flow) = self.flow {`?

Comment (inashivb) [reply rust/src/dcerpc/dcerpc.rs:623]: yes

**Comment by inashivb** [comment]
Replaced w #5902

**Pull request closed without merging**